tests/js/spec/views/organizationDiscover/discover.spec.jsx

```diff
@@ -515,30 +515,6 @@ describe('Discover', function() {
         })
       );
     });
-
-    it('changes date correctly', async function() {
```

Comment:
Moved this test to the container level so that we have a bigger integration test.